File: lab2/personClass/person.py
class Person:
    def __init__(self, name, first_name, date_of_birth, age):
        self._name = name
        self._first_name = first_name
        self._date_of_birth = date_of_birth
        self._age = age

    def __str__(self):
        return self._first_name + ' ' + self._name

File: cppCode/cppCode.d/Person.cpp

#include "Person.h"

Person :: Person (string nName, string nLast_name, string nDate_of_birth, int nAge){
	name = nName;
	last_name = nLast_name;
	date_of_birth = nDate_of_birth;
	age = nAge
}

string Person::get_name(){
	return name;
}

string Person::get_lname(){
	return last_name;
}

string Person::get_dob(){
	return date_of_birth;
}

int Person::get_age(){
	return age;
}
	
Person :: ~Person ()
	{ }
